src/js/app.js

```diff
@@ -29,8 +30,17 @@ const app = {
         $('#todos').append($todoContainer);
         
         app.counter = app.counter + 1;
+
+        $todoCheckbox.click(function () {
+            if (this.checked == true) {
```

Comment:
¿Qué cambios podrías hacer en esta condición para mejorarla?

Reply:
No igualarla a true, ya que this.checked es igual a true y entonces es redundante la condición al colocarle el true. Bien podría no tener nada o compararlo con un false. También es una buena idea colocar el operador idéntico para evitar errores.